perf/perf.js

```diff
@@ -9,6 +9,7 @@ import {
   benchmarkSubscribeSetup,
 } from './replicache.js';
 import {benchmarkIDBRead, benchmarkIDBWrite} from './idb.js';
+import {benchmarks as lockBenchmarks} from './lock.ts';
```

Comment:
if these are now running with the esbuild plugin we should remove all the extensions from the import paths for consistency.

Reply:
Done.